openstudio-geb.gemspec

```diff
@@ -22,11 +22,14 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency 'bundler', '~> 2.1'
+  spec.required_ruby_version = '~> 3.2.2'
+
+  spec.add_dependency 'openstudio-extension', '~> 0.8.0'
+  spec.add_dependency 'openstudio-standards', '0.6.1'
+
+  spec.add_development_dependency 'bundler', '~> 2.4.10'
   spec.add_development_dependency 'rake', '~> 13.0'
```

Comment:
I think we can also remove the requirements for `rake` and `bundler` for the same reason as above: they are included with extension-gem. In fact, I think we should remove them to prevent accidentally trying to use an incorrect version.